### ISSUE TITLE
fromBeginning option doesn't seem to be working

### ISSUE THREAD

**Issue by ChrisPenner** [lucagrulla/node-tail#39]
I'm not entirely sure what fromBeginning is supposed to do, and there don't seem to be tests for it; but what I'm trying to do is have the `tail.on('line', callback)` be fired once for each line in a given file when the `on` is first registered, then call it once for each following line that is appended.

Here's what I've got:
```javascript
        for (const filename of files) {
            let t = new Tail(filename, {fromBeginning: true});
            // Each time a new line comes in, send it along
            t.on("line", function(line){
                console.log(line);
                callback(line);
            });
        }
```

Passing the option doesn't seem to be changing anything at all, could someone please explain how it's supposed to work? Thanks!

**Comment by lucagrulla** [comment]
```fromBeginning``` tells the library to do exactly what you are looking for, i.e. start tailing from the beginning of the file rather than the first line appended. But what you highlighted is an actual bug, thanks for that. I'll try to release a new version with the bug fix over the next few days.

**Comment by nathankunicki** [comment]
Thanks for this, this is affecting me too. Any chance of publishing a new version so we can starting using it? :)

**Comment by lucagrulla** [comment]
fixed with v 1.0.1
